graphql/apollo_server/index.js

```diff
@@ -13,24 +13,25 @@ const axiosInstance = axios.create({
 const typeDefs = `#graphql
   
   type User {
-		id: Int!
-		name: String!
-		username: String!
-		email: String!
-		phone: String
-		website: String
-	}
+    id: Int!
+    name: String!
+    username: String!
+    email: String!
+    phone: String
+    website: String
+  }
 
-	type Post {
-		id: Int!
-		userId: Int!
-		title: String!
-		body: String!
-		user: User
-	}
```

Comment:
Please run prettier on all JS files. Use `\b\b` instead of `\t` for indentation everywhere.